### DISCUSSION TITLE
The Pillow… acronym?

### DISCUSSION THREAD

**Discussion by aclark4life** [python-pillow/Pillow#7992] [Polls]
At [this Pillow demo talk](https://www.youtube.com/watch?v=xVYqh5ElHUU&t=538s) I gave recently the host Jacob asked "What does Pillow stand for"? And I said "nothing" because it currently does not stand for anything. It was chosen due to it's similarity to the PIL acronym. At the time, I just needed another name that was close to the acronym PIL. 

Then I started thinking "If it were an acronym, what would it stand for?" The first thing I thought of was: The "**P**ython **I**maging **L**ibrary **L**ives **O**n **W**ell". That's OK but not great, so I asked ChatGPT and I got some interesting ideas. Of these, I have a favorite, but what do you think?

**Comment by hugovk** [comment]
>Should Pillow stand for something?

I vote for the missing option: "no, keep the word" :)

**Reply by aclark4life** [reply]
Right! If Discussions supported multiple poll questions I would have started with "Yes" or "No" …

**Reply by hugovk** [reply]
I didn't actually know discussions had polls before, so TIL :)

**Reply by aclark4life** [reply]
I don't really like Discussions, but I couldn't bring myself to open an Issue regarding this topic 😂
